lib/ansible/modules/cloud/azure/azure_rm_managed_disk.py

```diff
@@ -51,17 +51,17 @@
             - Premium_LRS
     create_option:
         description:
-            - "Allowed values: empty, import, copy. C(import) from a VHD file in I(source_uri) and C(copy) from previous managed disk I(source_resource_uri)."
+            - "Allowed values: empty, import, copy.
```

Comment:
hmm... I don't think we need additional description in case if we have choices defined. but that's minor.